.github/workflows/deploy.yml

```diff
@@ -40,10 +41,11 @@ jobs:
       - name: Build website
         run: |
           docker compose run --rm web /bin/bash -c "bundle check || bundle install --jobs=3"
-          docker compose run --rm web make build
+          docker compose run -e O --rm web make build
```

Comment:
```suggestion
          docker compose run -e $O --rm web make build
```

Reply:
No, that's correct. `-e O` means to pass the environment variable `O` into the container. It's short for `-e O=$O`.